storyblok/storyblok-design-system
# refactor: sb link to template syntax

## Pull request type

Jira Link: - 


## How to test this PR



Compare:
* https://blok.ink/?path=/story/design-system-components-sblink--default
* https://storyblok-design-system-git-refactor-ar-18-link-storyblok-com.vercel.app/

It's also good to check in the app:

* `cd storyblok-design-system && git checkout refactor/ar-18-link`
* Then start the server


## What is the new behavior?



- it uses the template api
- 
- 

## Other information
